experiments/process_sandbox/platform/sandbox.h

```diff
@@ -0,0 +1,40 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#pragma once
+
+/**
+ * This file contains the definition of a per-platform sandboxing policy.
+ * Because this interface currently has only one implementation, it is almost
+ * certainly the wrong abstraction and will change when others (e.g.
+ * seccomp-bpf) are added.
+ */
+
+#include "sandbox_capsicum.h"
+#include "sandbox_seccomp-bpf.h"
+
+namespace sandbox
+{
+  namespace platform
+  {
+    struct SandboxNoOp
+    {
+      template<typename T, typename U>
+      void restrict_file_descriptors(const T&, const U&)
```

Comment:
No comments whatsoever?

Reply:
Well, something that tells you this is for testing only? I don't know, this is not important.